src/Toggler.js

```diff
@@ -68,9 +68,11 @@ class Toggler extends State {
 			return content;
 		}
 
-		content = header.querySelector(this.content);
-		if (content) {
-			return content;
+		if (header.querySelector){
```

Comment:
Do you mean the same method on line #62? https://github.com/NolanChan/metal-toggler/blob/2bbbb887d52880665e1e09af1fc6104b6dd51405/src/Toggler.js#L62